src/lib/snack-bar/snack-bar-config.ts

```diff
@@ -19,4 +24,10 @@ export class MdSnackBarConfig {
 
   /** Extra CSS classes to be added to the snack bar container. */
   extraClasses?: string[];
+
+  /** The position to place the snack bar in the view, either 'left' or 'center'.  */
+  position?: MdSnackBarPosition = {
```

Comment:
I kind of want this to be two separate properties (`verticalPosition` and `horizontalPosition`)- it gives you the freedom to set just one if you want (easier defaults). Do you have a reason for making it a separate object?